flash/base.py

```diff
@@ -975,7 +975,7 @@ class RelatedModelInvalidationCache(object):
     """ Mixin class used in RelatedInstanceCache, RelatedQuerysetCache
     """
     def _get_invalidation_models(self):
-        return [self.model] + self.rel_models.keys()
+        return [self.model] + list(self.rel_models)
```

Comment:
[`dict.keys()` returned dict_keys object](https://stackoverflow.com/questions/18552001/accessing-dict-keys-element-by-index-in-python3)